tm2py/tools.py

```diff
@@ -0,0 +1,29 @@
+"""Tools module for common resources / shared code and "utilities" in the tm2py package."""
```

Comment:
`utils.py` is a bit more standard?  But fine with this if others are

Reply:
There was a specific concern raised with using "utils" / utilities" due to confusion with utilities in mode choice terminology.